server/app/blocks.go

```diff
@@ -356,14 +356,23 @@ func (a *App) notifyBlockChanged(action notify.Action, block *model.Block, oldBl
 		return
 	}
 
+	boardMember, _ := a.GetMemberForBoard(board.ID, modifiedByID)
+	if boardMember == nil {
+		// create temporary guest board member
+		boardMember = &model.BoardMember{
+			BoardID: board.ID,
+			UserID:  modifiedByID,
+		}
+	}
```

Comment:
Could you explain this? I don't understand the need for this. If the `modifiedByID` isn't a board member, wouldn't that be an error? Can that even happen? A mentioned user, may not be a member but I would think the modifier must be a member.

Reply:
My understanding is that if the user making the card edit is a guest, then `GetMemberForBoard` will return nil, since guests are not members.  However the APIs are expecting a `model.BoardMember` so I create a dummy one with no permissions.

Reply:
We aren't supporting guest accounts in v0.16. So (in theory), `boardMember` will never be `nil`, so I don't think we need to change it now.

But, just to be clear, when we add guest support, they must be explicitly added to the board according to this spec. 
https://docs.google.com/document/d/1hQAjl-4FFaEVCVYtU2BtyGxA97hcnNxCBDSnpnDT_cw/edit#bookmark=id.807067fo17yq